server/s3_option.cc

```diff
@@ -602,6 +604,8 @@ bool S3Option::load_section(std::string section_name,
              &libevent_pool_buffer_size);
       sscanf(libevent_max_read_size_str.c_str(), "%zu",
              &libevent_max_read_size);
+    } else if (section_name == "S3_VERSION_CONFIG") {
+      s3_version = s3_option_node["VERSION"].as<std::string>();
```

Comment:
This field cant be empty, it has to be there and it its empty then give error. We can take default version

Reply:
yes default is specified.